roles/matrix-cactus-comments/templates/systemd/matrix-cactus-comments.service.j2

```diff
@@ -0,0 +1,36 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Cactus Comments
+{% for service in matrix_bot_cactus_comments_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_bot_cactus_comments_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ matrix_systemd_unit_home_path }}"
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} kill matrix-cactus-comments 2>/dev/null || true'
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} rm matrix-cactus-comments 2>/dev/null || true'
+
+ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-cactus-comments \
+			--log-driver=none \
+			--cap-drop=ALL \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			--read-only \
+			--env-file {{ matrix_cactus_comments_app_service_env_file }} \
+            --mount type=bind,src={{ matrix_cactus_comments_container_tmp_path }},dst=/tmp \
```

Comment:
What is this used for? Do these temporary files really need to be persisted to the filesystem or can we use a tmpfs here?

Example: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/5f9f891322effa21d81c468cdcde701b6ba8c72c/roles/matrix-dendrite/templates/dendrite/systemd/matrix-dendrite.service.j2#L30

Reply:
No they do not I just didn't know better ;) Thanks!